tiup/customized-montior-in-tiup-environment.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Customize Configurations of Monitoring Servers
+summary: Learn how to customize the configurations of monitoring servers managed by TiUP
+---
+
+# Customize Configurations of Monitoring Servers
+
+When you deploy a TiDB cluster using TiUP, TiUP also deploys monitoring servers, such as Prometheus, Grafana, and Alertmanager. In the meantime, if you scale out this cluster, TiUP also adds the new nodes into monitoring scope.
+
+Noticeably, TiUP overwrites the configurations of the monitoring servers by using its configurations. That means, after you modify the configuration files of the monitoring servers, you might find that your modifications do not take effect because these modifications are overwritten by later TiUP operations such as deployment, scaling out, scaling in, and reloading.
+
+To customize the configurations of the monitoring servers mentioned above, you can follow the instructions below to add related configuration items in the topology.yaml of the TiDB cluster.
+
+> **Note:**
+>
```

Comment:
```suggestion
To customize the configurations of the monitoring servers mentioned above, you can follow the instructions below to add related configuration items in the topology.yaml of the TiDB cluster.

> **Note:**
>
> - When customizing the configurations of the monitoring servers, do not modify the configurations directly, because these modifications will be overwritten by later TiUP operations such as deployment, scaling out, scaling in, and reloading.

```